src/components/Contentful/StoryCard.vue

```diff
@@ -172,7 +172,6 @@ export default {
 		display: flex;
 		flex-direction: column;
 		height: 100%;
-		justify-content: center;
```

Comment:
I think the default of 'flex-start' looks better here, 'center' is creating some issues, see for example the misalignment of the "Evgeniya" card on: https://www.dev.kiva.org/lp/own-the-change-gender-equality